src/ccl_power.c

```diff
@@ -1432,7 +1432,7 @@ static double ccl_power_extrapol_highk(ccl_cosmology * cosmo, double k, double a
     
 }
 
-/*------ ROUTINE: ccl_power_extrapol_hxighk ----- 
+/*------ ROUTINE: ccl_power_extrapol_lowk ----- 
 INPUT: ccl_cosmology * cosmo, a, k [1/Mpc]
 TASK: extrapolate power spectrum at low k
 */
```

Comment:
Maybe worth changing the `kmin` argument to this function to `kmin_spline`.  I believe this parameter is what you now call K_MIN_SPLINE in the paper.  

It took me a little bit of tracking to follow how your explanations in the paper translated into code.  But my conclusion (correct me if I'm wrong) is that this function's `kmin` parameter is really the value that you refer to as K_MIN_SPLINE now.  It's value is (as you describe in the paper) different for different use cases.  E.g. CLASS vs EMU vs linear will have different lower limits of the spline arrays.  Then these lower limits get saved as either `cosmo->data.k_min_lin` or `cosmo->data.k_min_nl`, which in turn get passed to this function as the kmin argument.  

I think this connection could be made more explicit by calling this `kmin_spline` here and where you set the different `k_min_*` values just add an in-line comment saying that this is the K_MIN_SPLINE value for this particular choice of power spectrum being splined.

Reply:
Changed `kmin->kmin_spline` in `extrapol_lowk` and `kmax->kmax_spline` in `extrapol_highk`. The comment has been added.